src/main/java/org/myrobotlab/service/interfaces/ServoControl.java

```diff
@@ -29,10 +29,11 @@
 import java.util.Set;
 
 import org.myrobotlab.framework.interfaces.Attachable;
+import org.myrobotlab.framework.interfaces.StateSaver;
 import org.myrobotlab.math.Mapper;
 import org.myrobotlab.service.interfaces.ServoData.ServoStatus;
 
-public interface ServoControl extends AbsolutePositionControl, Attachable {
+public interface ServoControl extends AbsolutePositionControl, Attachable, StateSaver, org.myrobotlab.framework.interfaces.StatePublisher {
 

```

Comment:
do we need the fully qualified class name here? why not just import?  do we have a class name conflict with StatePublisher?

Reply:
yup